tests/benchmarks/variable_length_filter.yml

```diff
@@ -0,0 +1,23 @@
+name: "VARIABLE-LENGTH-FILTER"
+remote:
+  - setup: redisgraph-r5
+  - type: oss-standalone
```

Comment:
Can this be the default ?

Reply:
Yes, but not right away. We would need to add it to `defaults.yml` and I need to enable specifying the default remote env on the tool ( right now it only allows to inherit `kpis` and `exporter`.